scripts/conversion_tools/convert_pytorch_transformers.py

```diff
@@ -0,0 +1,221 @@
+# coding: utf-8
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# 'License'); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# pylint:disable=redefined-outer-name,logging-format-interpolation
+""" Script for converting the distilbert model from pytorch-transformer to Gluon.
+
+Usage: 
+
+pip3 install pytorch-transformers
```

Comment:
That package seems not maintained anymore. Why not convert from transformers package?

Reply:
At the time of writing this script, there was only the pytorch-transformer package. I think it's still useful to some users, who started to use bert with `pytorch-transformers`.